app/components/ExportPackModal/ThirdStep.js

```diff
@@ -83,10 +83,11 @@ const ThirdStep = props => {
         ]
       },
       manifestType: 'minecraftModpack',
+      overrides: "overrides",
       manifestVersion: 1,
-      version: config.modpackVersion,
-      author: config.projectID ? data.primaryAuthorName : props.username,
-      projectID: config.projectID,
+      version: props.packVersion,
+      author: config.projectID ? data.authors[0].name : props.username,
```

Comment:
Why `authors[0].name` instead of `primaryAuthorName`? @TOLoneWolf

Reply:
primary author didn't exist in the response from curse.

Reply:
examples.
modpack:
https://addons-ecs.forgesvc.net/api/v2/addon/285109
mod:
https://addons-ecs.forgesvc.net/api/v2/addon/238222